jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -4,7 +4,7 @@ pipeline {
             label 'community-build-executor'
         }
     }
-    
+    triggers { cron('0 22 * * 1,3,5') }
```

Comment:
Just to confirm this is not a daily cron schedule. Please check

Reply:
@shailesh-vaidya , I have updated the title and description of the PR. Pankaj requested for > https://github.com/Seagate/cortx-re/pull/1131#pullrequestreview-1041760530